crates/ra_hir_def/src/db.rs

```diff
@@ -107,6 +107,13 @@ pub trait DefDatabase: InternDatabase + AstDatabase {
     // Remove this query completely, in favor of `Attrs::docs` method
     #[salsa::invoke(Documentation::documentation_query)]
     fn documentation(&self, def: AttrDefId) -> Option<Documentation>;
+
+    #[salsa::invoke(crate::find_path::importable_locations_in_crate_query)]
+    fn importable_locations_in_crate(
```

Comment:
Not sure if this needs to be a query, doesn't feel like a hot-spot. Let's start with just an  utility funciton